### PR TITLE
Document the repo (code + readme)

### PR DESCRIPTION
**Pull request by Eleonore9**
Add docstrings, comments + update Readme.

### CODE REVIEW

File: src/witan/models/household.clj [hunk 1]
```diff
@@ -1,9 +1,12 @@
-(ns witan.models.household
+(ns ^{:doc "Defines the functions for each step of the household model.
+            The functions are defined using macros that make it easy to
+            be packaged into a datastructure to be run by a workspace executor"}
```
Comment (sunnytownsend) [inline src/witan/models/household.clj:3]: this is interesting - what does it do?

File: test/witan/models/household_test.clj [hunk 2]
```diff
@@ -59,14 +43,33 @@
...
+(def test-inputs {:input-resident-popn :resident-popn
+                  :input-institutional-popn :institutional-popn
+                  :input-household-rates :household-rates
+                  :input-vacancy-rates :vacancy-rates
+                  :input-second-homes-rates :second-homes-rates})
+
+(defn read-inputs [file schema]
+  {})
+
+(defn fix-input
```
Comment (sunnytownsend) [inline test/witan/models/household_test.clj]: the name `fix-input` is a bit confusing to me - what do you mean by fix here?

File: src/witan/models/household.clj (hunk 1 above)
Comment (Eleonore9) [reply src/witan/models/household.clj:3]: I think it's to add a docstring to the namespace

File: test/witan/models/household_test.clj (hunk 2 above)
Comment (Eleonore9) [reply test/witan/models/household_test.clj]: I think I re-used a name Antony was using in the demography test. I'll change that!

**Comment by sunnytownsend** [comment]
:shipit: lgtm

**Pull request merged**